manifests/init.pp

```diff
@@ -91,15 +91,11 @@
   String $jvm_xmx                                                   = '1024m',
   String $jvm_permgen                                               = '256m',
   Optional[String] $jvm_optional                                    = undef,
-  Optional[String] $jvm_optional_additional                         = undef,
   Optional[String] $jvm_extra_args                                  = undef,
-  Optional[String] $jvm_extra_args_additional                       = undef,
   Optional[String] $jvm_gc_args                                     = undef,
-  Optional[String] $jvm_gc_args_additional                          = undef,
   Optional[String] $jvm_codecache_args                              = undef,
-  Optional[String] $jvm_codecache_args_additional                   = undef,
   Integer $jvm_nofiles_limit                                        = 16384,
-  String $java_opts                                                 = '',
+  Optional[String] $java_opts                                       = undef,
```

Comment:
I think it doesn't make sense to pass `''` to the variable, so we could update it to this?
```suggestion
  Optional[String[1]] $java_opts                                       = undef,
```

Reply:
Empty `$java_opts` is perfectly valid currently, so that would be an API break. Since it gives a deprecation warning anyway, I think it's better to accept whatever configuration people have now.

Reply:
okay, fine by me